Riot/Managers/Theme/Themes/DefaultTheme.swift

```diff
@@ -105,6 +105,14 @@ class DefaultTheme: NSObject, Theme {
     
     var roomCellOutgoingBubbleBackgroundColor: UIColor = UIColor(rgb: 0xE7F8F3)
     
+    var roomCellLocalisationTextColor: UIColor = UIColor(rgb: 0x17191C)
+    
+    var roomCellLocalisationStartedColor: UIColor = UIColor(rgb: 0x5C56F5)
```

Comment:
We can maybe precise icon: `roomCellLocalisationIconStartedColor` or use a generic `liveLocationSharingIconStartedColor`